e2e/linter/src/linter.test.ts

```diff
@@ -500,9 +492,12 @@ describe('Linter', () => {
           {
             "dependencies": {
               "@nx/devkit": "${nxVersion}",
+              "tslib": "^2.3.0",
```

Comment:
This is not correct, we need to use the above deleted `tsLibVersion` from `dependencies` (not `devDependencies`).